config/config-builder/src/main.rs

```diff
@@ -51,6 +52,9 @@ enum FullNodeCommand {
 
 #[derive(Debug, StructOpt)]
 struct FullNodeArgs {
+    #[structopt(long)]
+    /// Defines which chain version libra will use
+    chain_id: Option<u8>,
```

Comment:
why not use the string representation?

Reply:
because then we have to use a fixed subset, which is antithetical to the entire goal of using the chain id for networks....

Reply:
fwiw the numeric value as a string also works (e.g. "2" and "TESTNET" both work)

Reply:
Let's just get this working for now and we can screw around with the semantics later!